src/debug/debuggerConfigureDriver.ts

```diff
@@ -0,0 +1,26 @@
+import { CMakeOutputConsumer } from "@cmt/diagnostics/cmake";
+import * as vscode from "vscode";
+
+export const debuggerPipeName = "\\\\.\\\\pipe\\\\cmake-debugger-pipe";
```

Comment:
Using a constant name for the pipe makes it impossible to have multiple CMake debug sessions running at the same time.

Reply:
Fixing.